elyra/pipeline/airflow/package_catalog_connector/airflow-package-catalog.json

```diff
@@ -60,7 +60,8 @@
           "type": "string",
           "format": "uri",
           "uihints": {
-            "category": "Source"
+            "category": "Source",
+            "ui:placeholder": "http://host:port/path/apache_airflow.whl"
```

Comment:
did not get to this until now, sorry. In the future, maybe you can change the placeholder to https, as is mentioned in the documentation with regards to pythonhosted and as most e.g. Artifactory users will use. Other than that, very good idea with the UI placeholders.